L1Trigger/Phase2L1ParticleFlow/plugins/L1TCorrelatorLayer1Producer.cc

```diff
@@ -380,14 +393,25 @@ void L1TCorrelatorLayer1Producer::produce(edm::Event &iEvent, const edm::EventSe
     //l1pualgo_->runNeutralsPU(l1region, z0, -1., puGlobals);
   }
 
+  // FIXME: what about STA objects? we might need them if we use the per board output to write in CMSSW (to get the ref)
```

Comment:
?

Reply:
Ops, ignore it. The ref problem is now solved by splitting the put operation. I will soon commit a new version